### PR TITLE
Only create postgres_db if not using external

### PR DESCRIPTION
**Pull request by joao-p-marques**
Fixes https://github.com/Tecnativa/doodba-copier-template/issues/148

### CODE REVIEW

File: common.yaml.jinja [hunk 1]
```diff
@@ -32,6 +32,7 @@ services:
       traefik.port: "8069"
     {%- endif %}
 
+  {%- if postgres_version is not none %}
```
Comment (Yajo) [inline common.yaml.jinja:35]: You can simplify this, here and in all others:

```suggestion
  {%- if postgres_version %}
```

File: devel.yaml.jinja [hunk 2]
```diff
@@ -69,13 +69,15 @@ services:
       - --dev=reload,qweb,werkzeug,xml
       {%- endif %}
 
+  {%- if postgres_version is not none %}
```
Comment (Yajo) [inline devel.yaml.jinja:72]:
```suggestion
  {%- if postgres_version %}
```

File: prod.yaml.jinja [hunk 3]
```diff
@@ -45,13 +45,15 @@ services:
       ) }}
       {%- endif %}
 
+  {%- if postgres_version is not none %}
```
Comment (Yajo) [inline prod.yaml.jinja:48]:
```suggestion
  {%- if postgres_version %}
```

File: test.yaml.jinja [hunk 4]
```diff
@@ -47,13 +47,15 @@ services:
       - --workers=2
       - --max-cron-threads=1
 
+  {%- if postgres_version is not none %}
```
Comment (Yajo) [inline test.yaml.jinja:50]:
```suggestion
  {%- if postgres_version %}
```

**Review by Yajo — CHANGES_REQUESTED** [review]

**Comment by ska-ibees** [comment]
odoo@ubuntu20:~/Docker/dock13$ invoke img-pull
Reinitialized existing Git repository in /home/odoo/Docker/dock13/.git/
pre-commit installed at .git/hooks/pre-commit
Service 'odoo' depends on service 'db' which is undefined.

**Comment by ska-ibees** [comment]
Can we have 'DATABASE_URL' in copier question to get some flexibility? Like I intend to deploy Doodba on AWS EC2 by connecting it to AWS RDS database (Over network).

**Pull request merged**